### PR TITLE
Introduce WIP state

### LINKED ISSUE #16: New Feature Suggestion: `WIP` on title can move the story to a state(like `In Development`)
When PR opens, I think it usually means to want to get reviews for their code. (usually ready for review?)
If we write a title with `WIP`, I think the meaning is different from the case above.
So I suggest a routine that if `WIP` on the title, the shortcut story card moves to a specific workflow state.

@anencore94 What do you think about it? if good, I wanna contribute by posting PR for this routine.
thanks :)

### PR DESCRIPTION
**Pull request by hunhoon21**
This PR will close #16 

## description
* introduce wip workflow state(like in development) id and wip keyword(like WIP) to move shortcut story card to WIP state

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -11,6 +11,13 @@
 from pydantic import BaseSettings, Field
 
 
+def parse_wip_on_title(title: str, wip_keyword: str) -> bool:
+    regexp = re.compile(f"{wip_keyword}", re.IGNORECASE)
```
Comment (anencore94) [inline src/main.py:15]: maybe we could remove f-string

File: src/main.py [hunk 2]
```diff
@@ -57,6 +64,8 @@ class Settings(BaseSettings):
     shortcut_api_token: str = Field(..., env="INPUT_SHORTCUT_API_TOKEN")
     open_id: str = Field(..., env="INPUT_PR_OPENED")
     closed_id: str = Field(..., env="INPUT_PR_CLOSED")
+    wip_id: str = Field(env="INPUT_PR_WIP", default="")
+    wip_keyword: str = Field(env="INPUT_WIP_KEYWORD", default="")
```
Comment (anencore94) [inline src/main.py:68]: we need to add these field to `action.yaml` and README.md

**Review by anencore94 — COMMENTED** [review]
Thanks for you contribution :)

File: src/main.py (hunk 1 above)
Comment (hunhoon21) [reply src/main.py]: I've removed that

File: src/main.py (hunk 2 above)
Comment (hunhoon21) [reply src/main.py:68]: I updated that part!

**Comment by hunhoon21** [comment]
Sorry for fixing late for your feedback. I've changed them. Thank you :)

File: src/main.py (hunk 2 above)
Comment (anencore94) [reply src/main.py:68]: Thanks!

**Review by anencore94 — APPROVED** [review]
LGTM!

**Pull request merged**